lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -215,4 +216,11 @@ defmodule ExUnit.CLIFormatter do
       _ -> 80
     end
   end
+
+  defp print_logs(""), do: nil
+
+  defp print_logs(output) do
+    output = String.replace(output, "\n", indent = "\n     ")
```

Comment:
Can you assign `indent` outside of the function call,  it is not immediately obvious where indent comes from on the next line.